cf_xarray/accessor.py

```diff
@@ -1593,10 +1622,15 @@ def get_associated_variable_names(
         names : dict
             Dictionary with keys "ancillary_variables", "cell_measures", "coordinates", "bounds".
         """
-        keys = ["ancillary_variables", "cell_measures", "coordinates", "bounds"]
+        keys = [
+            "ancillary_variables",
+            "cell_measures",
+            "coordinates",
+            "bounds",
+            "grid_mapping",
+        ]
         coords: dict[str, list[str]] = {k: [] for k in keys}
         attrs_or_encoding = ChainMap(self._obj[name].attrs, self._obj[name].encoding)
```

Comment:
```suggestion
        attrs_or_encoding = ChainMap(self._obj[name].attrs, self._obj[name].encoding)

```

It's nice to minimize any unnecessary changes.